deluge/ui/console/modes/connectionmanager.py

```diff
@@ -174,28 +170,21 @@ def on_resize(self, rows, cols):
     def read_input(self):
         c = self.stdscr.getch()
 
-        if c > 31 and c < 256:
-            if chr(c) == 'q' and self.inlist:
-                return
+        if is_printable_chr(c):
```

Comment:
`is_printable_chr` tests for value `c >= 32 and c <= 126`, so this doesn't test for the same as the original code. I'm not sure what input values are > 126 though.